testproto2/proto2.letmegrpc.go

```diff
@@ -652,6 +653,34 @@ func (this *htmlProto2) Produce(w net_http.ResponseWriter, req *net_http.Request
 	jsonString := req.FormValue("json")
 	someValue := false
 	msg := &Album{}
+	validateMap := make(map[string]interface{})
+	err := encoding_json.Unmarshal([]byte(jsonString), &validateMap)
+	if err != nil {
+		log.Printf("validator: %s ", err.Error())
```

Comment:
I think this is just an unmarshal error, the validator string is just confusing things to the user?

Reply:
You're right, but , I think the server side user should know if there is a bad json request from client side user, so, I use the `log` here . It will probably produce some spams but for debugging it should be considered here.

Reply:
No I am not contesting the log :)
I am still saying we can simply print out the error instead of prepending "validator:" or come up with a more descriptive string to prepend the error with.

Reply:
sry for misunderstanding, I'll remove this prefix.